src/styles/_catalog.scss

```
@@ -1,22 +1,28 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  box-sizing: border-box;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

тут repeat точно не потрібний
```suggestion
  grid-template-columns: 200px;
```